doc/source/whatsnew/v0.22.0.txt

```diff
@@ -320,5 +320,10 @@ Categorical
 Other
 ^^^^^
 
+- Improved error message when attempting to use a Python keyword as an identifier in a numexpr query (:issue:`18221`)
+- Fixed a bug where creating a Series from an array that contains both tz-naive and tz-aware values will result in a Series whose dtype is tz-aware instead of object (:issue:`16406`)
```

Comment:
you have rebase issues here

Reply:
Will fix.  Not a big deal, but for changes limited to whatsnew is skipping the CI a viable option?  The CI turnaround time is close to a full day and that file gets touched a lot.